raiden_contracts/contract_manager.py

```diff
@@ -186,7 +198,14 @@ def get_contracts_deployment_info(
             services=False,
         ))
 
-    if module == DeploymentModule.SERVICES or module == DeploymentModule.ALL:
+    if module == DeploymentModule.SERVICES and not version_provides_services(version):
+        raise ValueError(
+            f'SERVICES module queried for version {version}, but {version}'
+            'does not provide service contracts.',
+        )
+
+    if (module == DeploymentModule.SERVICES or module == DeploymentModule.ALL) and \
```

Comment:
style nitpick:

We never use `\` to prolong a line in raiden. Better use an extra variable for readability

```python
module_match = module == DeploymentModule.SERVICES or module == DeploymentModule.ALL
if module_match and version_provides_services(version):
    file.append(....
```

Also wouldn't the check `if version_provides_services(version)` be enough here?

Reply:
> wouldn't the check if version_provides_services(version) be enough here?

When people ask for `module=RAIDEN`, the service contracts should not be included in the result.

Reply:
ah yes. My bad!